hcp-consul-hcp-vault-ca/variables.tf

```
@@ -0,0 +1,79 @@
+variable "region" {
+  default     = "us-east-1"
+  description = "Region in which to run this terraform code for the AWS Provider"
+}
+variable "node_group_configuration" {
+
+  default = {
+    ami_type       = "AL2_x86_64"
+    instance_types = ["m5.large"]
+    disk_size_gigs = 50
+  }
+}
+
```

Add a variable for a customizable cluster name.
```suggestion
variable "cluster_name" {
  type = string
  default = "learn-hcp"
  description = "Name to be used on all the resources as identifier."
}
```